app/src/main/java/org/jellyfin/androidtv/ui/AlphaPicker.kt

```diff
@@ -0,0 +1,41 @@
+package org.jellyfin.androidtv.ui
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.widget.Button
+import android.widget.LinearLayout
+import androidx.core.view.children
+import org.jellyfin.androidtv.R
+
+class AlphaPicker(
+	context: Context,
+	attrs: AttributeSet? = null
+) : LinearLayout(context, attrs) {
+	var onAlphaSelected: (letter: Char) -> Unit = {}
+
+	init {
+		removeAllViews()
```

Comment:
I don't think this is needed because `init` is basically the constructor body so there shouldn't be any child views.